src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs

```diff
@@ -160,10 +165,9 @@ internal void SetForeground(object localValue)
 					break;
 				case GradientBrush gradient:
 					this.SetStyle(
-						("background", gradient.ToCssString(this.RenderSize)),
-						("color", "transparent"),
-						("background-clip", "text")
-					);
+						("--before-background", gradient.ToCssString(this.RenderSize)),
```

Comment:
Should we prefix CSS variables with `uno-`?  Like `--uno-before-background`.  What do you think?

Reply:
@carldebilly I don't have a strong opinion here.

Note: https://github.com/unoplatform/uno/pull/3410 will need to follow that if we decided to prefix with uno. cc @MartinZikmund 

Let me know if you decided to prefix.